.travis.yml

```diff
@@ -177,7 +208,7 @@ script:
     ./configure CXXFLAGS="-W -Wall -Wextra -Werror -Wno-unused-parameter -Wno-maybe-uninitialized -Wno-unknown-warning-option"
     make
   elif [[ "$TRAVIS_BUILD_SYSTEM" == "Makefile" ]]; then
-    make -f Makefile.simple
+    $mingw64 make -f Makefile.simple
```

Comment:
I had to hack this line a bit, but I think it's fine, since for non-Windows builds `$mingw64` is just empty.

Reply:
Here, does `make` get expanded out via `export MAKE=mingw32-make` from line 159? I'm guessing not?
I don't like relying on unset env vars to be empty; could this just be conditional on Windows, i.e.,

    elif [[ "$TRAVIS_BUILD_SYSTEM" == "Makefile" && "$TRAVIS_OS_NAME" == "windows" ]]; then
        $MINGW64 make -f Makefile.simple
    elif [[ "$TRAVIS_BUILD_SYSTEM" == "Makefile" ]]; then
        make -f Makefile.simple